package.json

```diff
@@ -101,23 +99,23 @@
     "bundlesize": [
         {
             "path": "./packages/melody-component/lib/index.js",
-            "maxSize": "1.10 kB"
+            "maxSize": "1.16 kB"
         },
         {
             "path": "./packages/melody-idom/lib/index.js",
-            "maxSize": "4.67 kB"
+            "maxSize": "5.6 kB"
```

Comment:
Do you have an idea on why this happens and how to prevent it?

Reply:
I am looking into it. 🔍  I changed the uglify with terser. It could be one reason.